resources/telemetry/charts/operator/templates/deployment.yaml

```diff
@@ -30,6 +30,7 @@ spec:
           args:
             - --leader-elect
             - --cm-name={{ .Release.Name }}-{{ .Values.logProcessor }}-sections
+            - --parser-cm-name={{ .Release.Name }}-{{ .Values.logProcessor }}-parsers
```

Comment:
We should align the parsers volume mount in the fluent-bit chart values with this one. Also double-check the other ConfigMaps.